packages/fxa-payments-server/package.json

```diff
@@ -14,8 +14,8 @@
     "build": "react-scripts build",
     "eject": "react-scripts eject",
     "test": "npm-run-all test:*",
-    "test:frontend": "react-scripts test --coverage --verbose",
-    "test:server": "jest --coverage --verbose --config server/jest.config.js",
+    "test:frontend": "react-scripts test --coverage --verbose --transformIgnorePatterns 'node_modules/(?!speed-trap)/'",
```

Comment:
I have published speed-trap 0.0.10 so you shouldn't have to special case this anymore.